bokeh/models/glyphs.py

```diff
@@ -893,6 +926,39 @@ class Text(Glyph):
     The %s values for the text.
     """)
 
+class VBar(Glyph):
+    """ Render vertical bars, given a center coordinate, width and (top, bottom) coordinates. """
+
+    __example__ = "tests/glyphs/VBar.py"
+
+    # a canonical order for positional args that can be used for any
+    # functions derived from this class
+    _args = ('x', 'width', 'top', 'bottom')
+
+    x = NumberSpec(help="""
+    The x-coordinates of the centers of the bars.
+    """)
+
+    width = NumberSpec(help="""
+    The widths of the vertical bars.
+    """)
+
+    bottom = NumberSpec(help="""
+    The y-coordinates of the bottom edges.
+    """)
```

Comment:
we could be opinionated here, it's possible to add `default=0` and that would make the `bottom=0` unnecessary for bars that do happen to start at 0